### ISSUE TITLE
[DOC]: cuSPatial users guide (and notebook) shows version 22.08 in conda command line

### ISSUE THREAD

**Issue by harrism** [rapidsai/cuspatial#1238]
### How would you describe the priority of this documentation request

Medium

### Is this for new documentation, or an update to existing docs?

Update

### Describe the incorrect/future/missing documentation

https://docs.rapids.ai/api/cuspatial/stable/user_guide/cuspatial_api_examples/

```
# !conda create -n rapids-22.08 -c rapidsai -c conda-forge -c nvidia \ 
#     cuspatial=22.08 python=3.9 cudatoolkit=11.5 
```

This uses cuspatial 22.08 -- we should update the `update-version.sh` script to also update the `cuspatial_api_examples.ipynb` so that we update this to the latest release each release.

### If this is a correction, please provide a link to the incorrect documentation. If this is a new documentation request, please link to where you have looked.

https://docs.rapids.ai/api/cuspatial/stable/user_guide/cuspatial_api_examples/

**Comment by jarmak-nv** [comment]
I think this should get resolved after we merge #1232 https://github.com/rapidsai/cuspatial/blob/78e7685922802b28736d5af7a2ba0f94e93bb80e/ci/release/update-version.sh#L90

**Comment by harrism** [comment]
I think I added this to #1239 or #1217
